releasenotes/releasenotes_v2.rst

```diff
@@ -247,7 +247,6 @@ Feature
 #. The plugin folder (:ref:`settings_pipeline`) may now contain subfolders. Plugins will be read from all underlying folders.
 #. Vonk supports If-Match on update. See `Managing Resource Contention <http://hl7.org/fhir/http.html#concurrency>`_ in the specification for details.
 #. Plugins may return non-FHIR content. See :ref:`vonk_plugins_directhttp`.
-#. This feature may also be used for :ref:`accesscontrol_custom_authentication`.
```

Comment:
Should this not be re-routed to accesscontrol_api.rst?

Reply:
No, I removed the content completely. It was not something that we should promote in general.